GoogleDAIDemo/app/src/main/java/com/jwplayer/googledai/MainActivity.java

```diff
@@ -0,0 +1,95 @@
+package com.jwplayer.googledai;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.longtailvideo.jwplayer.JWPlayerView;
+import com.longtailvideo.jwplayer.configuration.PlayerConfig;
+import com.longtailvideo.jwplayer.events.AdErrorEvent;
+import com.longtailvideo.jwplayer.events.listeners.AdvertisingEvents;
+import com.longtailvideo.jwplayer.media.ads.dai.ImaDaiSettings;
+import com.longtailvideo.jwplayer.media.playlists.PlaylistItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity
+        implements AdvertisingEvents.OnAdErrorListener{
+
+    JWPlayerView mPlayerView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        mPlayerView = findViewById(R.id.jwplayer);
+
+        List<PlaylistItem> playlist = new ArrayList<>();
+        //videoId: Identifier of the DAI video to be displayed, used for video on demand
+        //cmsId: Content management system ID of the video, used for video on demand
+        String videoId = "bbb-clear";
+        String cmsId = "2474148";
+        //streamType: Defines the type of stream to use
+        ImaDaiSettings.StreamType streamType = ImaDaiSettings.StreamType.DASH;
+        ImaDaiSettings imaDaiSettings = new ImaDaiSettings(videoId, cmsId, streamType, null);
+        //fallbackUrl: Contain URL in case ads stream fails. Since the content is a stream
+        //you must be ready to call a fallback stream if the DAI stream encounters an error
```

Comment:
This comment implies the developer has to do extra work if the stream fails, but we handle that automatically.  The comment should instead read something like
//fallbackUrl: Contain URL in case ads stream fails. This url
//will automatically be used in case the DAI stream encounters an error